content/admin/enterprise-management/monitoring-using-snmp.md

```diff
@@ -66,7 +66,7 @@ If you enable SNMP v3, you can take advantage of increased user based security t
 
 ##### Querying SNMP data
 
-Both hardware and software-level information about your appliance is available with SNMP v3. Due to the lack of encryption and privacy for the `noAuthNoPriv` and `authNoPriv` security levels, we exclude the `hrSWRun` table (1.1.3.6.1.2.1.25.41) from the resulting SNMP reports. We include this table if you're using the `authPriv` security level.
+Both hardware and software-level information about your appliance is available with SNMP v3. Due to the lack of encryption and privacy for the `noAuthNoPriv` and `authNoPriv` security levels, we exclude the `hrSWRun` table (.1.3.6.1.2.1.25.4) from the resulting SNMP reports. We include this table if you're using the `authPriv` security level.
```

Comment:
I think this is indeed a mistake in the current version of this article, but the leading `.` should probably removed, too. 
I also wonder whether we should link to http://oidref.com/1.3.6.1.2.1.25.4, in order to make this a bit clearer?